src/Swarm/Game/State.hs

```diff
@@ -474,6 +483,10 @@ robotsAtLocation loc gs =
     . view robotsByLocation
     $ gs
 
+-- | Get a list of all the robots that are "watching" any location.
+-- This is the counterpart to the per-robot "watchedLocations" member.
+robotsWatching :: Lens' GameState (S.Set RID)
```

Comment:
Should I even bother with having this as a performance optimization?  Or is it fine to just iterate over all extant robots to obtain this information?

Reply:
Changed to `Map Location (Set RID)`.